src/AddAggregatesPlugin.ts

```diff
@@ -239,6 +288,80 @@ const AddAggregatesPlugin: Plugin = builder => {
         },
         {}
       ),
+      ...pgIntrospectionResultsByKind.procedure.reduce(
+        (memo: GraphQLFieldConfigMap<any, any>, proc: PgProc) => {
+          /* TODO: Do fields need to be omitted? */
+          const attrIsNumberLike =
+            pgIntrospectionResultsByKind.typeById[proc.returnTypeId]
+              .category === "N";
+          if (attrIsNumberLike && pgColumnFilter(proc, build, context)) {
+            const computedColumnDetails = getComputedColumnDetails(
+              build,
+              table,
+              proc
+            );
+            if (computedColumnDetails) {
+              const fieldName = inflection.computedColumn(
+                computedColumnDetails.pseudoColumnName,
+                proc,
+                table
+              );
+              return build.extend(memo, {
+                [fieldName]: pgField(
+                  build,
+                  fieldWithHooks,
+                  fieldName,
+                  ({ addDataGenerator }: any) => {
+                    addDataGenerator((parsedResolveInfoFragment: any) => {
+                      return {
+                        pgQuery: (queryBuilder: QueryBuilder) => {
+                          // Note this expression is just an sql fragment, so you
+                          // could add CASE statements, function calls, or whatever
+                          // you need here
+                          const expr = sql.fragment`${sql.identifier(
+                            proc.namespaceName,
+                            proc.name
+                          )}(${queryBuilder.getTableAlias()})`;
+                          queryBuilder.select(
+                            // You can put any aggregate expression here; I've wrapped it in `coalesce` so that it cannot be null
+                            sql.fragment`coalesce(sum(${expr}), 0)`,
+                            // We need a unique alias that we can later reference in the resolver
+                            getSafeAliasFromAlias(
+                              parsedResolveInfoFragment.alias
+                            )
```

Comment:
This dynamic reference _is_ required, because if you accept args later you'd need to be able to specify the field multiple times:

```graphql
calc1: computedColumn(a: 1)
calc2: computedColumn(a: 2)
```

Reply:
(If it weren't required then you could simplify with a static alias. But you can't because it is required.)